### PR TITLE
Add reverse functionality

### PR DESCRIPTION
**Pull request by NagaMavullammaGundu**
This branch adds reverse functionality and its testcases to the main branch.

### CODE REVIEW

File: my_functions.py [hunk 1]
```diff
@@ -39,4 +39,9 @@ def is_prime(num):
     for i in range(2, int(num**0.5) + 1):
         if num % i == 0:
             return False
-    return True
\ No newline at end of file
+    return True
+
+def reverse_string(s):
```
Comment (phani1995) [inline my_functions.py:44]: Add a doc string explaining the function, its arguments and its return values.

File: test_my_functions.py [hunk 2]
```diff
@@ -94,3 +94,24 @@ def test_prime(inputnumber, expectedOutput):
...
+
+
+def test_reverse_float():
+    with pytest.raises(TypeError, match=r"Input must be a string"):
+        result = my_function.reverse_string(100.879)
+
+
+def test_reverse_negative():
+    with pytest.raises(TypeError, match=r"Input must be a string"):
+        result = my_function.reverse_string(-100)
```
Comment (phani1995) [inline test_my_functions.py]: These three function testcases can be parameterized.

File: my_functions.py (hunk 1 above)
Comment (NagaMavullammaGundu) [reply my_functions.py:44]: Added doc string

File: test_my_functions.py (hunk 2 above)
Comment (NagaMavullammaGundu) [reply test_my_functions.py]: Did it.

File: my_functions.py (hunk 1 above)
Comment (phani1995) [reply my_functions.py:44]: Nice

File: test_my_functions.py (hunk 2 above)
Comment (phani1995) [reply test_my_functions.py]: Good

**Pull request merged**